app/js/adminPage.ts

```diff
@@ -100,10 +100,15 @@ document.querySelector('#addNewUserForm').addEventListener('submit', function(ev
     let timeTotal = convertToTotalTimeSeconds(timeMinutes, timeSeconds);
 
     getExistingUsers().then(function(existingUsers) {
+        let letters = /^[A-Za-z]+$/;
```

Comment:
this regex doesn't allow spaces in the name
![image](https://user-images.githubusercontent.com/25844355/131876030-a08bab0c-f898-4bb7-a937-64912c8a5d0e.png)

Reply:
Updated